synfig-studio/src/gui/widgets/widget_link.h

```diff
@@ -46,12 +46,23 @@ class Widget_Link: public Gtk::ToggleButton
 	Gtk::Image *icon_off_;
 	Gtk::Image *icon_on_;
 
+	void init();
```

Comment:
why two `init()`?
What about the constructors that would call `init()` actually call `init("", "")` instead? :)

Reply:
```c++
...
Widget_Link::Widget_Link():
	Glib::ObjectBase("widget_link")
{
	init();
}

Widget_Link::Widget_Link(
		const string &tlt_inactive,
		const string &tlt_active):
	Glib::ObjectBase("widget_link")
{
	init(tlt_inactive, tlt_active);
}
...
```
In default constructor, there are no string parameters. How would it call `init("", "")`..?

Reply:
As far as I can see, `init()` and `init(const string&, const string&)` are equal, except the former doesn't set the tooltips.
So it calls that way, without any tooltips. Am I wrong?

Reply:
Yes and it's need for wrap_new

Reply:
This is also related to Glib::Property I think. I won't be fixing this too.

You can enhance it in the future :)